core/languages.lua

```diff
@@ -1,5 +1,9 @@
 local loadkit = require("loadkit")
-local cldr = require("cldr")
+
+-- BEGIN OMIKHLEIA HACKLANG
+-- Disabled for now, see further below.
+--   local cldr = require("cldr")
```

Comment:
See comment below, if ICU actually does what we need here that's fine, otherwise cldr-lua should be extended to provide the necessary functionality.